packages/components/src/ColumnManager/ColumnManager.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { Button, DataList, Modal, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import ColumnManagerListItem from './ColumnManagerListItem';
+
+const ColumnManager = ({ isOpen, columns, onSave: onSaveArgument, onClose: onCloseProp, selectedColumns: selectedColumnsProp = [] }) => {
+  const [selectedColumns, setSelectedColumns] = useState(selectedColumnsProp);
+  const isSelectedColumn = ({ title }) => selectedColumns.includes(title);
```

Comment:
The `isSelectedColumn` function is moved into a helpers module.

The `title` is really not the best. It's now changed to use the key.

Additionally the key is a required prop for the column prop.
If no key is given it will not show up in the manager, this is also to encourage using a key prop for columns.